src/main/resources/templates/admin.html

```diff
@@ -32,8 +32,8 @@ <h1 class="text-center text-md-left">UH Groupings Administration</h1>
     <div class="container">
         <div id="overlay" ng-show="loading">
             <div class="d-flex justify-content-center">
-                <div class="spinner-border text-light" style="width: 7rem; height: 7rem; margin-top: 17em;" ng-show="loading"></div>
-            </div>
+            <div class="spinner-border text-light spinner-style" ng-show="loading"></div>
+                </div>
```

Comment:
Could you fix the spacing here? After that, everything in this PR is good to go.